src/languages/sass.coffee

```diff
@@ -2,7 +2,6 @@ module.exports = {
 
   name: "Sass"
   namespace: "sass"
-  fallback: ['css', 'scss']
```

Comment:
The `fallback` field is for options fallback handling. For example, if the user has not explicitly set options for `sass` then Atom-Beautify can use options configured for `css` as a fallback. This allows developers to configure for `css`, `html`, `javascript`, etc and not have to worry about explicitly configuring  languages that are similar yet different such as `scss`, `sass`, `less`, `handlebars`, `jade`, `typescript`, `flow`, etc.

And so I would at least prefer to keep `fallback: ['css']`. Let me know what you think.